spec/commands/menu/set_menu_spec.rb

```diff
@@ -1,45 +1,44 @@
-require 'spec_helper'
 require 'commands/menu/set_menu_command'
+require 'models/apprentice'
 require 'models/menu'
+require 'spec_helper'
 
 RSpec.describe SetMenuCommand do
-  let (:menu) { SetMenuCommand.new }
-  let (:fake_data) {{user_message: "www.menu.com", user_id: "id"}}
-
   before(:each) do
     foreman = Apprentice.new(
       user_name: "Will",
-      slack_id: "id"
+      slack_id: "valid id"
     )
     foreman.save
   end
 
   it "we can change the default url" do
```

Comment:
I think that description doesn't read as nice as it could be. What's the behaviour you're describing here of `it`?